File: 0268-missing-number/0268-missing-number.cpp
class Solution {
public:
    int missingNumber(vector<int>& nums) {
        int N = nums.size();
        int summation = (N * (N + 1)) / 2;

        // Summation of all array elements:
        int s2 = 0;
        for (int num : nums) {
            s2 += num;
        }

        int missingNum = summation - s2;
        return missingNum;
        
    }
};

File: 0268-missing-number/0268-missing-number.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var missingNumber = function(nums) {
    const N = nums.length;
    const summation = (N * (N + 1)) / 2;

    // Summation of all array elements:
    const s2 = nums.reduce((acc, num) => acc + num, 0);

    const missingNum = summation - s2;
    return missingNum;
};